README.md

```diff
@@ -17,6 +17,12 @@
   - Follow the prompts in the installer (Accept the license agreement, click the NEXT button and accept the default installation settings).
   ##### For Mac users
   - use homebrew/nvm to install node 
+2. You have to know the basic of JavaScript. 
+  - dofactory: JavaScript reference guide http://www.dofactory.com/tutorial/javascript
+  - W3School: https://www.w3schools.com/js/default.asp

```

Comment:
I would never recommend w3school, as it has a lot of wrong information and it is not trustable resource - it is a lot of information about it in web.